test/check/components.go

```diff
@@ -38,7 +38,7 @@ var (
 		ClusterRoleBinding:         "nmstate-handler",
 		ClusterRole:                "nmstate-handler",
 		SecurityContextConstraints: "nmstate",
-		DaemonSets:                 []string{
+		DaemonSets: []string{
```

Comment:
Needs go formatting

Reply:
It's correct, sorry. Look ugly, but this was go fmt that did it; it's consistent w/ https://github.com/kubevirt/cluster-network-addons-operator/pull/346/files#diff-a0979fa1282fb419073a584875cb4b30R50